### PR TITLE
Move chain setup

### LINKED ISSUE #317: refactor: move chain setup out to private package
The `cosmic-swingset/setup` directory should be moved to the `testnet` folder.  This is for cleanliness and in anticipation of chains other than Cosmos.

### PR DESCRIPTION
**Pull request by michaelfig**
Closes #317

### CODE REVIEW

**Comment by michaelfig** [comment]
This has been tested by build and deployment to my personal testtestnet.

File: packages/cosmic-swingset/Makefile [hunk 1]
```diff
@@ -63,7 +60,7 @@ scenario2-setup:
 	echo 'mmmmmmmm' | $(AGC) gentx --home-client=t1/$(BASE_PORT)/ag-cosmos-helper-statedir --name=ag-solo --amount=1000000stake
 	$(AGC) collect-gentxs
 	$(AGC) validate-genesis
-	./setup/set-json.js ~/.ag-chain-cosmos/config/genesis.json --agoric-genesis-overrides
+	../deployment/set-json.js ~/.ag-chain-cosmos/config/genesis.json --agoric-genesis-overrides
```
Comment (warner) [inline packages/cosmic-swingset/Makefile:63]: This is the only bit that worries me, having this package be dependent upon something outside the package subdirectory. But I can't think of a clean solution for it right now, so I guess let's land it this way and then worry about it later.

File: yarn.lock [hunk 2]
```diff
@@ -126,6 +126,11 @@ ajv@^6.10.0, ajv@^6.10.2:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+ansi-escapes@^3.2.0:
```
Comment (warner) [inline yarn.lock:129]: Why did yarn.lock change? Does the new package have a different set of dependencies than the old one?

**Review by warner — APPROVED** [review]

File: yarn.lock (hunk 2 above)
Comment (michaelfig) [reply yarn.lock:129]: The old dependencies weren't tracked by yarn before... the package.json was in `packages/cosmic-swingset/setup`.

Comment (warner) [reply yarn.lock:129]: ah, thanks

File: packages/cosmic-swingset/Makefile (hunk 1 above)
Comment (michaelfig) [reply packages/cosmic-swingset/Makefile:63]: As we do the refactorings in #263 and #317, we can address this better.

**Pull request merged**